sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusAmqpConnection.java

```diff
@@ -63,4 +74,11 @@
      */
     Mono<ServiceBusReceiveLink> createReceiveLink(String linkName, String entityPath, ReceiveMode receiveMode,
         String transferEntityPath, MessagingEntityType entityType, String sessionId);
+
+    /**
+     * Gets the transaction manager for a Service Bus resource and performs transaction related operations using it.
+     *
+     * @return A Mono that completes with that resources's management node.
+     */
+    Mono<TransactionChannel> createChannel();
```

Comment:
We don't have a concept of a channel in Service Bus... why would we need transaction channel?

Reply:
This was taken from RabbitMQ , but I removed it now and using TransactionManager.